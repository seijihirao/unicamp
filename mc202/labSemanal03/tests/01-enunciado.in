i 0 1.2
i 1 2.3
i 2 3.4
i 3 4.5
i 4 5.6
i 5 6.7
i 6 7.8
i 7 8.9
p
v 1 5
p
v 4 7
p
t
