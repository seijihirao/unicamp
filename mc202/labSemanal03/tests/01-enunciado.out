1.2000 2.3000 3.4000 4.5000 5.6000 6.7000 7.8000 8.9000 
1.2000 6.7000 5.6000 4.5000 3.4000 2.3000 7.8000 8.9000 
1.2000 6.7000 5.6000 4.5000 8.9000 7.8000 2.3000 3.4000 
